test/suites/auth.sh

```diff
@@ -102,6 +102,13 @@ test_authorization() {
   # The token was used, so the pending identity should be deleted.
   [ "$(lxc auth identity list --format csv | grep -cF 'pending')" = 0 ]
 
+  # Check token prune task works
+  lxc auth identity create tls/test-user2 --quiet
+  sleep 2
```

Comment:
why sleep 2?

Reply:
To wait for the token to expire. The expiry is still set to 1 second. I'll add a comment.